### ISSUE TITLE
[Docs] vaex integration: create flytesnacks example page

### ISSUE THREAD

**Issue by cosmicBboy** [flyteorg/flyte#3285]
### Description

The Vaex integration currently does not have an example page under the integrations section: https://github.com/flyteorg/flytesnacks/tree/master/cookbook/integrations/flytekit_plugins

The purpose of this task is to add (a) a page describing the plugin and how to install it (see [here](https://docs.flyte.org/projects/cookbook/en/latest/auto/integrations/flytekit_plugins/pandera_examples/index.html)) and (b) an example page on how to use it (see [here](https://docs.flyte.org/projects/cookbook/en/latest/auto/integrations/flytekit_plugins/pandera_examples/basic_schema_example.html#sphx-glr-auto-integrations-flytekit-plugins-pandera-examples-basic-schema-example-py))

### Are you sure this issue hasn't been raised already?

- [X] Yes

### Have you read the Code of Conduct?

- [X] Yes

**Comment by cosmicBboy** [comment]
TODO: reach out to OS community for contribution

**Comment by TechWizard9999** [comment]
😊 "I'd be delighted to work on this issue, @cosmicBboy ! Could you please assign it to me?"

**Comment by samhita-alla** [comment]
@TechWizard9999, are you working on creating a PR for this issue?

**Comment by priyavratamohan** [comment]
Hi, @samhita-alla, can you assign this issue to me?

**Comment by mertbozkir** [comment]
Hello @samhita-alla and @cosmicBboy, I created an example readme file for Vaex integration, but the Python code is still waiting to be implemented. 

Probably be solved until Tuesday! 🤘

**Comment by Murdock9803** [comment]
Hello @davidmirror-ops, I will be really glad to get the chance to work on this issue. Please assign the issue to me 🙌 

Also, the links to resources given in the issue description about `Installing and Using the Plugin` redirect to a `404 error-not found` page. Please guide about that too 🙌 ❤️

**Comment by davidmirror-ops** [comment]
@Murdock9803 right. This is because those pages refer to integrations that are now implemented using the Agents framework. So describing the integration and how to install it ([reference](https://docs.flyte.org/en/latest/deployment/agents/databricks.html)) and an example of how to use it ([reference](https://docs.flyte.org/en/latest/flytesnacks/examples/databricks_agent/index.html)).
As you may see, these two sources are on the [flytesnacks repo](https://github.com/flyteorg/flytesnacks), feel free to tag me as reviewer once you have the PR. Thanks!
